tuxemon/core/states/combat/combat.py

```diff
@@ -694,6 +710,9 @@ def end_combat(self):
         fadeout_action.parameters = [1000]
         event_engine.actions["fadeout_music"]["method"](self.game, fadeout_action)
 
+        #Round player movement in case battle started in the middle of a step
+        self.game.player1.tile_pos = (round(self.game.player1.tile_pos[0]),round(self.game.player1.tile_pos[1]))
```

Comment:
Yes, that's fine. I mainly just want to prevent fixing one bug while introducing another.